lib/xlat_tables_v2/aarch32/xlat_tables_arch.h

```diff
@@ -51,13 +51,13 @@
 
 # error "PLAT_VIRT_ADDR_SPACE_SIZE is too big."
 
-#elif PLAT_VIRT_ADDR_SPACE_SIZE > (1 << L1_XLAT_ADDRESS_SHIFT)
+#elif PLAT_VIRT_ADDR_SPACE_SIZE > (1ULL << L1_XLAT_ADDRESS_SHIFT)
```

Comment:
Is this really required? L1_XLAT_ADDRESS_SHIFT is 30 , so 1 << 30 is still within unsigned int range. Perhaps `1U` would suffice ?